packages/pdfx/lib/src/viewer/pinch/pdf_view_pinch.dart

```diff
@@ -310,7 +310,8 @@ class _PdfViewPinchState extends State<PdfViewPinch>
       return;
     }
     _forceUpdatePagePreviews = false;
-    for (final page in _pages) {
+    for (var i = 0; i < _pages.length; i++) {

```

Comment:
For what?

Reply:
When I open a pdf and close it fastly, it throws an error that this for is iterating a list that is empty, because when the page gets disposed, the _pages list gets cleared.